tests/expected/c/10083-infinite-loops.c

```diff
@@ -0,0 +1,83 @@
+int main(void)
+{
+	int x;
+
+	do
+	{
+	}while(true);
+
+	do
+	{
+	}while(true);
+
+	do
+	{
+	}
+	while (true);
+
+	do
+	{
+	}while(true);
+
+	do
+	{
+	}
+	while(true);
+
+	do
+		x++; while(true);
```

Comment:
I think we could add a new like before while(true) when replacing into do-while loop. That should be an easy change and probably what everyone would expect, regardless of what settings is used for other options.